test/machines/machines_test.go

```diff
@@ -96,19 +146,207 @@ var _ = framework.SigKubeDescribe("Machines", func() {
 			f.CreateClusterAndWait(cluster)
 
 			// Create/delete a single machine, test instance is provisioned/terminated
-			uri := "qemu:///system"
-			testMachineProviderConfig, err := utils.TestingMachineProviderConfig(uri, cluster.Name)
+			testMachineProviderConfig, err := utils.TestingMachineProviderConfig(f.LibvirtURI, cluster.Name)
 			Expect(err).NotTo(HaveOccurred())
 			testMachine := manifests.TestingMachine(cluster.Name, cluster.Namespace, testMachineProviderConfig)
-			libvirtClient, err := NewLibvirtClient(uri)
+			lcw, err := NewLibvirtClient("qemu:///system")
```

Comment:
why static libvirt address?

Reply:
The e2e binary is run inside the same instance as the libvirt is [1]. Thus just the local connection.

[1`] https://github.com/openshift/aos-cd-jobs/pull/1623/files#diff-f415831f3c4220b99bc2fd31df494224R79